src/collections/_documentation/workflow/releases/index.md

```diff
@@ -334,3 +334,74 @@ You can also use our [API]({%- link _documentation/api/releases/post-release-dep
 ## Release Artifacts
 
 JavaScript and iOS projects can utilize release artifacts to unminify or symbolicate error stack traces. To learn more, please check out our [iOS]({%- link _documentation/clients/cocoa/index.md -%}#sentry-cocoa-debug-symbols) and [JavaScript]({%- link _documentation/platforms/javascript/sourcemaps.md -%}) docs.
+
+## Release Health
+
+{% capture __alert_content -%}
+Sentry’s Release Health features are currently in beta. For more details about access to these features, feel free to reach out at [LINK: email?].
+{%- endcapture -%}
+{%- include components/alert.html
+    title="Note"
+    content=__alert_content
+    level="warning"
+%}
+
+Monitor the health of releases by observing user adoption, usage of the application, percentage of [crashes]({%- link _documentation/workflow/releases/release-health-glossary.md -%}#crash), and [session data]({%- link _documentation/workflow/releases/release-health-glossary.md -%}#session). Release health will provide insight into the impact of crashes and bugs as it relates to user experience, and reveal trends with each new issue through the release details, graphs, and filters.
+
+### Releases Index
+
+The releases index page gives a high-level view of each release version, its associated project, the authors of each commit, the percentage of crash-free users, and the percentage of crash-free sessions. Each release links to one project. If a release has multiple projects, Sentry will duplicate the release data in relation to each unique project. The data will present adoption of the release and usage in the most recent 24 hours to 14 days.
```

Comment:
Can we mention in here that when the time range is changed the metrics are recalculated and link to the FAQ?